<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>com</class>
 <widget class="QMainWindow" name="com">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>com</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_18">
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>RS485 LINK</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>串口:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>波特率:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>数据位:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>校验位:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>停止位:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QComboBox" name="portNameComboBox"/>
             </item>
             <item row="1" column="0">
              <widget class="QComboBox" name="baudRateComboBox">
               <item>
                <property name="text">
                 <string>9600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>14400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>19200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>38400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>56000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>57600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>76800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>115200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>128000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>256000</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QComboBox" name="dataBitsComboBox">
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QComboBox" name="parityComboBox">
               <item>
                <property name="text">
                 <string>wu</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ji</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ou</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QComboBox" name="stopBitsComboBox">
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QPushButton" name="openBtn">
             <property name="text">
              <string>打开串口</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="closeBtn">
             <property name="text">
              <string>关闭串口</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>CMD</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="1">
              <widget class="QLineEdit" name="cmdText"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>CMD:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="sendBtn">
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QPushButton" name="resetBtn">
             <property name="text">
              <string>系统复位</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="stopBtn">
             <property name="text">
              <string>急停</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="changeBtn">
             <property name="text">
              <string>换笔</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="zeroBtn">
             <property name="text">
              <string>回零</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Circle</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_24">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_15">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>X(mm):</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="xText">
             <property name="text">
              <string>375</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="1">
            <widget class="QLineEdit" name="yText">
             <property name="text">
              <string>250</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Y(mm):</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>R(mm):</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="rtext">
             <property name="text">
              <string>150</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_16">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_21">
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Start:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="startText">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_22">
           <item row="0" column="1">
            <widget class="QLineEdit" name="sideText">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Side:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <layout class="QGridLayout" name="gridLayout_23"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="testBtn">
         <property name="text">
          <string>启动</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>DEBUG</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_20">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_19">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_17">
           <item row="0" column="4">
            <widget class="QLCDNumber" name="recvbyteslcdNumber"/>
           </item>
           <item row="0" column="7">
            <widget class="QPushButton" name="clearUpBtn">
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>计数：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="chradioButton">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>十六进制形式显示</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="ccradioButton">
             <property name="text">
              <string>字符形式显示</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QTextBrowser" name="receiveText"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
